packages/bee-q/src/components/breadcrumb/bq-breadcrumb.tsx

```diff
@@ -98,9 +98,9 @@ export class BqBreadcrumb {
 
   render() {
     return (
-      <Host role="list">
+      <nav class="breadcrumb" role="list">
```

Comment:
You could have left the Tailwind CSS classes, they don't get applied only in the `<Host>` tag.

Reply:
Also a shadow part is missing here 🙂